operations/package_repository_management/Nexus_Space_Cleanup.md

```diff
@@ -12,20 +12,23 @@ This page outlines the procedure to manually cleanup Nexus, in order to ensure t
 
 - [Cleanup of data not being used](#cleanup-of-data-not-being-used)
 - [Cleanup of old installs](#cleanup-of-old-installs)
+- [Remove Data Marked for Deletion](#remove-data-marked-for-deletion)
+- [Check PVC Size](#check-pvc-size)
 - [Increase PVC size](#increase-pvc-size)
 
 ## Cleanup of data not being used
 
-Any data in Nexus that is not currently being used can be deleted to make space for the upgrade. This can include data added after the
-previous install, or data that was added during an install that is no longer needed. If there is anything in Nexus that is no longer
-needed, then it is recommended to delete that first, before taking any further steps.
+Any data in Nexus that is not currently being used can be deleted to make space for the upgrade. Data includes repositories or artifacts,
+to delete a blob store refer to [Remove Data Marked For Deletion](#remove-data-marked-for-deletion).This can include data added after the
```

Comment:
```suggestion
To delete a blob store, refer to [Remove Data Marked For Deletion](#remove-data-marked-for-deletion). This can include data added after the
```